ui/raidboss/common_replacement.js

```diff
@@ -10,7 +10,7 @@ let commonReplacement = {
     ko: '15초 후에 $1(이|가) 봉쇄됩니다',
   },
   'is no longer sealed': {
-    de: 'öffnet sich erneut',
+    de: 'öffnet sich wieder',
```

Comment:
What about both?

Reply:
i can make it both tbh but in logmessages:2014 it is `wieder`. but will put both in just to be sure